BWEM/include/area.h

```diff
@@ -173,6 +177,8 @@ class Area : public utils::Markable<Area, int>, public utils::UserData
 	std::vector<Mineral *>			m_Minerals;
 	std::vector<Geyser *>			m_Geysers;
 	std::vector<Base>				m_Bases;
+
+	mutable std::vector<BWAPI::Position>	m_boundaryVertices;
```

Comment:
I guess this `mutable` relates to the `const` question I have above, if the above is changed, this should be changed too.